packages/graphql-dynamodb-transformer/src/resources.ts

```diff
@@ -160,7 +180,20 @@ export class ResourceFactory {
             },
             Policies: [
                 new IAM.Role.Policy({
-                    PolicyName: 'DynamoDBAccess',
+                    PolicyName: Fn.If(
```

Comment:
You don't need to modify policy names. Only role names can't clash. There can be policies with the same name under different IAM roles.

Reply:
Makes sense. I'll revert this.